### ISSUE TITLE
Handling @id references that point to URL

### ISSUE THREAD

**Issue by marcolarosa** [Arkisto-Platform/describo-online#10]

**Comment by marcolarosa** [comment]
Example:

```
{
 additionalType: [ { @id: 'http://something' } ]
}
```

This example points to a URL and not something in the graph so need to add a condition in the crate loading library that inserts a special entity:

```
{
  @type: 'URL',
  @id: 'http://something'
}
```

when it encounters these references.


See `api/src/lib/crate.js` lines 232-244 - add insert entity method conditional with associate.

**Comment by marcolarosa** [comment]
closed by 02a2e2209a330006eacd96333fe98f8615506554
